src/Microsoft.ML.AutoML/ColumnInference/ColumnInferenceApi.cs

```diff
@@ -3,13 +3,16 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using Microsoft.ML.Data;
 
 namespace Microsoft.ML.AutoML
 {
     internal static class ColumnInferenceApi
     {
+        private const int _maxUngroupedColumns = 20000;
```

Comment:
What is this for?

Reply:
setting the threshold value for grouping columns? Model Builder doesn't use that.